src/aqua/remove_worker.aqua

```diff
@@ -0,0 +1,118 @@
+module RemoveWorker declares remove
+export remove, inspect, joined_deals, get_logs, test_join, test, remove_all
+
+import Worker from "@fluencelabs/aqua-lib/workers.aqua"
+import Srv, Op from "@fluencelabs/aqua-lib/builtin.aqua"
+import Spell, StringListValue from "@fluencelabs/spell/spell_service.aqua"
+
+import getWorkersInfo, GetWorkersInfoHostsDeciderInstallationSpells from "workers.aqua"
```

Comment:
I can't find workers.aqua and this type `GetWorkersInfoHostsDeciderInstallationSpells` (what a name huh)

Reply:
you have to do `fluence build` first